wagtailio/static/sass/components/_feature.scss

```diff
@@ -0,0 +1,64 @@
+.feature {
+    $root: &;
+    display: flex;
+    align-items: flex-start;
+    gap: 10px;
+
+    &__link,
+    &__text {
```

Comment:
suggestion: Could we add this to the parent class?

Reply:
Yeah - good shout! 😀